MSAL/src/native_auth/controllers/credentials/MSALNativeAuthCredentialsController.swift

```diff
@@ -91,10 +91,10 @@ final class MSALNativeAuthCredentialsController: MSALNativeAuthTokenController,
             let config = factory.makeMSIDConfiguration(scopes: [])
             return try cacheAccessor.getAllAccounts(configuration: config)
         } catch {
-            MSALLogger.log(
+            MSALLogger.logPII(
                 level: .error,
                 context: nil,
-                format: "Error retrieving accounts \(error)")
+                format: "Error retrieving accounts \(MSALLogMask.maskPII(error))")
```

Comment:
this error can contains EUII, can you mark it as such?

Reply:
Done, thank you